.github/ISSUE_TEMPLATE/05-release.md

```diff
@@ -32,7 +32,15 @@ production readiness.
 
 - [ ] Choose the commit for the release. The commit should *always* be the
   last commit before noon EST on Monday, even if you start the release process
-  late.
+  late. From an updated `main` branch, use this command to find the release commit:
+
+  ```shell
+  # If you run the release on Monday
```

Comment:
Did you test these on both macOS and Gnu date?

Reply:
No, just gnu. Do they work on your Mac?

Reply:
Classic:

```
$ date -Iminutes -d 'this monday 12:00 EST'
date: illegal option -- I
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```

Reply:
We can just say to use gdate if you are on macOS. People can install it with brew probably.

Reply:
confirmed, `brew install coreutils` gives you gdate on macOS

Reply:
Thanks, added the note